pydtk/db/schemas/dataware-tools.com/v1alpha2/record.py

```diff
@@ -15,10 +15,13 @@ class Record(BaseSchema):
     record_id: constr(min_length=1) = Field(..., description="")
 
     # Allow additional properties
-    class Config:
+    class Config(BaseSchema.Config):
         """Config."""
 
         extra = Extra.allow
-        schema_extra = {
-            "additionalProperties": {}
-        }
+
+        @staticmethod
+        def schema_extra(schema: Dict[str, Any], model: Type['Record']):
+            """Extra schema."""
+            BaseSchema.Config.schema_extra(schema, model)
```

Comment:
継承しているクラスの関数を使っていることがわかりやすくなるように

```suggestion
            super().schema_extra(schema, model)
```

にしても良いかと思いました (これだと動かないかもですが)

Reply:
super() だと
```TypeError: super(type, obj): obj must be an instance or subtype of type```
となって動かず...
https://stackoverflow.com/questions/26788214/super-and-staticmethod-interaction
↑も試してみたんですがダメでした

Reply:
なるほど。。であればこのままでOKです！